src/components/ElasticQueryField/ElasticQueryField.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TextArea from '@folio/stripes-components/lib/TextArea';
+
+const defaultStyle = {
```

Comment:
 Please remove inline styles. I believe currents TextArea styles are ok for POC. If in future we would need to change it I belive we would need to add the ability for TextArea to extend base styles.

Reply:
Thanks for the feedback, fixed.